MoneySplitter.Services/API/ApiUrlBuilder.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using MoneySplitter.Infrastructure;
+
+namespace MoneySplitter.Services.Api
+{
+    public  class ApiUrlBuilder : IApiUrlBuilder
+    {
+        public Uri Authorization()
+        {
+            return new Uri(string.Concat(
+                Defines.Api.WEB_API_URL,
+                Defines.Api.Session.SESSION,
+                Defines.Api.Session.SIGN_IN));
+        }
+
+        public Uri Registration()
+        {
+            return new Uri(string.Concat(
+                Defines.Api.WEB_API_URL,
+                Defines.Api.Session.SESSION,
+                Defines.Api.Session.REGISTR));
```

Comment:
registEr, please rename anywhere this word

Reply:
Rename